src/api/fetchServerEmojiData.js

```diff
@@ -70,15 +70,15 @@ export default async (emojiDataUrl: URL): Promise<ServerEmojiData> => {
 
   const httpStatus = response.status;
   if (httpStatus >= 400 && httpStatus <= 499) {
-    // Client error…?
+    // Client error…? Server error…?
```

Comment:
Ah, what I think this line was trying to say is that (a) an HTTP status of 4xx specifically means a client error (not a server error), but (b) because it's not clear how that would come about, it seems just as likely to be a server issue.